### ISSUE TITLE
Crashing on packing empty strings

### ISSUE THREAD

**Issue by Fuyukai** [ccubed/Earl#11]
```
In [2]: earl.pack({'aa': ''})
fish: “ipython” terminated by signal SIGSEGV (Address boundary error)
```

**Comment by ccubed** [comment]
Fixed in 475b89c2d4ce94bc208b610808916246e4e3a575

**Comment by ccubed** [comment]
I guess to be more clear the code that fixes this is in 13fd713e7adc7b85385eef446e68894959fd1f19
